packages/web3-common/src/eth_execution_api.ts

```diff
@@ -20,9 +23,8 @@ export interface AccessListEntry {
 export type AccessList = AccessListEntry[];
 export type TransactionHash = HexString;
 export type Uncles = HexString32Bytes[];
-export type BlockTag = 'earliest' | 'latest' | 'pending';
-export type BlockNumberOrTag = Uint | BlockTag;
```

Comment:
Renamed and moved to [web3-utils](https://github.com/ChainSafe/web3.js/blob/c7933b378db2e220ca0444f3481e1d0c4cb16d04/packages/web3-utils/src/types.ts#L25)